src/types.ts

```diff
@@ -4,6 +4,13 @@ import { Request, Response } from 'express';
 import { IncomingMessage, ServerResponse } from 'http';
 import { History, Location } from 'history';
 
+export interface LayoutProps { 
+  location: Location; 
+  children: React.ReactNode 
+}
+
+export type LayoutComponent = React.ComponentClass<LayoutProps>
```

Comment:
Could we make this `ComponentType<LayoutProps>` so the layout can be stateless?

Reply:
Sure, but I need a proper review from @jaredpalmer before I'm gonna touch the code again.